operations/rack_resiliency/Enabling_Rack_Resiliency.md

```diff
@@ -7,6 +7,9 @@
 * By default, Rack Resiliency is disabled.
 * Rack Resiliency can be enabled only during fresh install of CSM 1.7 or an
   upgrade from CSM 1.6 to CSM 1.7.
+* This page documents the procedures for enabling and configuring Rack Resiliency during
+  a CSM install or upgrade. For information on how to do this on a running system, see
+  [Enable Rack Resiliency on a Running System](Enabling_RR_on_running_system.md).
 * Rack Resiliency cannot be disabled after it has been enabled during the install or upgrade.
 
 At the same time that Rack Resiliency is enabled, administrators also have the option
```

Comment:
This paragraph states that RR cannot be enabled later, which contradicts the rest of this PR.

The existing RR content needs to be carefully examined to make sure that it is consistent and accurate.

Reply:
This still needs to be addressed. The wording of this paragraph still indicates that the decision to enable RR cannot be changed later, but that is only partially true now -- you can choose to enable it later if you choose not to enable it during the install, but you cannot choose to disable it later if you choose to enable it during the install. So the wording here needs to be updated in some way.